tsconfig.json

```diff
@@ -10,7 +10,8 @@
     "sourceMap": true,
     "inlineSources": true,
     "strict": true,
-    "target": "es6"
+    "target": "es6",
+    "skipLibCheck": true, // Skip type checking of declaration files.
```

Comment:
during building the typescript compiler (while builds the package) give me a weird TS error on an internal / referenced package (ethereumjs). Indeed it's valid, the issue is in the typing of the library. Are you okay with switching this off? @gantunesr @mcmire

Reply:
Only as a last resort. This results in any type declarations defaulting to `any` if they have errors, so it decreases type safety and can let errors sneak in without us noticing.

What was the error you encountered?

Reply:
Getting this:
```
node_modules/@ledgerhq/metamask-keyring/node_modules/@ethereumjs/tx/node_modules/@ethereumjs/common/dist/types.d.ts:45:6 - error TS1336: An index signature parameter type cannot be a type alias. Consider writing '[key: string]: string | AccountState' instead.

45     [key: PrefixedHexString]: PrefixedHexString | AccountState;
        ~~~
```

Reply:
Gotcha - we have seen that error already in other repositories. I don't fully understand what's going on there, but we've found that the error goes away after a TypeScript update. I can look into updating TypeScript.

Reply:
That'd be great, I did try and didn't seem to resolve the issue.

Reply:
I have just confirmed locally that updating TypeScript fixes this issue. It introduced a number of other build issues, but those were all easily resolved and will be addressed as part of the update PR.

Reply:
@zoltanarvai since you were able to solve this issue it should be safe to revert this change